html/changelogs/Sindroman-airbubble.yml

```diff
@@ -0,0 +1,8 @@
+author: PoZe
+
+delete-after: True
+
+changes:
+  - bugfix: "Airbubble no longer produces infinite cable restrains."
+  - bugfix: "Users of airbubble no will not 'magically' get out of bubble with it still being closed."
+  - tweak: "Airbubble now doesn't have time to get in and out."
```

Comment:
Grammatically invalid, though I don't recall the exact rule. (have/take)
Perhaps: `Airbubbles no longer take time to enter/exit.`